Portfolio.css

```diff
@@ -1,3 +1,173 @@
+* {
+    box-sizing: border-box;
+}
+
 Body {
     font-family: 'Quicksand', sans-serif;
+    font-size: 62.5%;
+    background-color: #fff;
+    color: #5D6063;
+}
+
+.main-nav {
+    background-color: #fff;
+    color: #5D6063;
```

Comment:
Same inconsistent use of long and short hex values

Reply:
done.